bihire/team_work_ui
# Feature

### What does this PR do ?
- Admin can delete a flagged article
- Admin can delete a flagged comment
- responsive nav bar in all chrome devices
- Employee can delete their own Articles
- Employee can flag and comment on other people's articles
### How should this be manually tested ?
After cloning this repository
- go to UI directory open article_detail.html click report this article button
- shrink any page in the repository to check the responsiveness
- go to UI directory open article_detail.html click delete button
### What are the relevant pivot-tracker stories ?
[#168246247] delete a flagged comment & article
[#168332973] responsive navbar
### Screenshots
![image](https://user-images.githubusercontent.com/37307172/64855524-58493100-d620-11e9-9040-5605281ee0a1.png)
![image](https://user-images.githubusercontent.com/37307172/64855580-79aa1d00-d620-11e9-82e1-e7808b3432dc.png)
![image](https://user-images.githubusercontent.com/37307172/64855626-9c3c3600-d620-11e9-9d01-a374c9442b87.png)
